.pipeline/scripts/print-coverage.py

```
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import os
+import os.path
+import sys
+import yaml
+from glob import glob
+
+def get_jacoco_thresholds():
+    config = None
+    with open(".pipeline/config.yml", "r") as stream:
+        try:
+            config = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```

Doesn't that ignore the error. How does it continue without thresholds?